mystery_word.py

```diff
@@ -0,0 +1,265 @@
+
+from random import choice
+
+
+def word_pull():
+    """Pulls a list of words from the computer's dictionary"""
+    with open("/usr/share/dict/words") as file:
+        all_words = file.readlines()
+    return all_words
+
+
+def chooser(a_list):
+    """Chooses one word from a list of words"""
+    chosen_word = choice(a_list)
+    return chosen_word
+
+
+def set_chosen_word_string(chosen_word):
+    """Provides a clean, uppper-cased string"""
+    cleaned_word = chosen_word[:-1]
+    chosen_word_string = cleaned_word.upper()
+    return chosen_word_string
+
+
+def set_chosen_word_list(chosen_word_string):
+    """Creates list form of chosen_word_string"""
+    chosen_word_list = list(chosen_word_string[:])
+    return chosen_word_list
+
+
+def length_checker(a_string):
+    """Determines the length of a string"""
+    string_length = len(a_string[:-1])
+    return string_length
+
+
+def easy_sorter(a_list):
+    """Sorts the list of all words into words of 4-6 letters"""
+    easy_list = [x for x in a_list if len(x) > 4 and len(x) < 8]
+    return easy_list
+
+
+def med_sorter(a_list):
+    """Sorts the list of all words into words of 6-10 letters"""
+    med_list = [x for x in a_list if len(x) > 6 and len(x) < 12]
+    return med_list
+
+
+def hard_sorter(a_list):
+    """Sorts the list of all words into words of 10+ letters"""
+    hard_list = [x for x in a_list if len(x) > 10]
+    return hard_list
+
```

Comment:
These three functions are very similar. Here's a good place to create one function that all three call, instead of having replicated code in all three.